src/components/addButton.js

```diff
@@ -0,0 +1,22 @@
+import { useDispatch } from 'react-redux';
+import { addBooks } from '../redux/features/book/bookSlice';
+
+function AddButton() {
+  const dispatch = useDispatch();
+  const addNew = (e) => {
+    e.preventDefault();
+    const bookTitle = e.target[0];
+    const bookAuthor = e.target[1];
+    const itemId = Math.floor((Math.random() * 1000) + 1);
+    dispatch(addBooks({ title: bookTitle.value, author: bookAuthor.value, item_id: itemId }));
+  };
+  return (
+    <form className="bookForm" onSubmit={addNew}>
+      <input className="bookName" placeholder="bookName" />
+      <input className="authorName" placeholder="authorName" />
+      <button className="addButton" type="submit">Add</button>
+    </form>
+  );
```

Comment:
Nice work so far. However, according to the project requirements, you are supposed to use a button component  (<Button>) which include an event handler that adds a book to the books array. 
Since you have the addBooks function coming from the bookSlice, there is no need creating a new function (addNew) and using the onSubmit on the whole form. 
Just as you did in the Book component for the remove functionality, kindly consider adding the event listener (onClick) on the button component instead. eg: 
   ```jsx
      <Button onClick={() => dispatch(addBook({ title, author }))}>
        Add book
      </Button>
```